### PR TITLE
Epmlstrtai 794

### PR DESCRIPTION
**Pull request by nikandtos**
Here contains few completed tasks.  Changing roles(789), assign users to project(792-790) and CRUD for TEST_ENGINEER(794).

### CODE REVIEW

File: jdi-cucumber-test-generator-integration-tests/src/test/java/com/epam/test_generator/api/steps/then/Registration.java [hunk 1]
```diff
@@ -35,4 +40,13 @@ public void userIsCreated() throws Throwable {
             equalTo(expectedRegistrationUserDTO.getEmail()));
     }
 
-}
+    @Then("^I get bad status and warning$")
```
Comment (pomkine) [inline jdi-cucumber-test-generator-integration-tests/src/test/java/com/epam/test_generator/api/steps/then/Registration.java]: what warnings?

File: jdi-cucumber-test-generator-integration-tests/src/test/java/com/epam/test_generator/api/steps/utils/Basics.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+import com.epam.test_generator.controllers.user.response.UserDTO;
+import com.epam.test_generator.dto.TokenDTO;
+import org.springframework.http.HttpStatus;
+
+import static com.epam.http.requests.RequestData.requestBody;
+import static com.epam.test_generator.api.ApiTokenInserter.requestBodyAndToken;
+import static com.epam.test_generator.api.ApiTokenInserter.requestDataAndToken;
+import static com.epam.test_generator.api.BddGeneratorApi.*;
+
+public class Basics extends StepBackground {
```
Comment (pomkine) [inline jdi-cucumber-test-generator-integration-tests/src/test/java/com/epam/test_generator/api/steps/utils/Basics.java]: Why you need to extend `StepBackground` here?

Comment (pomkine) [inline jdi-cucumber-test-generator-integration-tests/src/test/java/com/epam/test_generator/api/steps/utils/Basics.java]: Please think about better class naming

File: jdi-cucumber-test-generator-integration-tests/src/test/java/com/epam/test_generator/api/steps/when/Registration.java [hunk 3]
```diff
@@ -15,6 +15,14 @@
 
     @When("^I register a new user$")
     public void sendUserDto(DataTable table) throws Throwable{
+//        RegistrationUserDTO registrationUserDTO = new RegistrationUserDTO();
```
Comment (pomkine) [inline jdi-cucumber-test-generator-integration-tests/src/test/java/com/epam/test_generator/api/steps/when/Registration.java]: pls remove it if we don't need it.

File: jdi-cucumber-test-generator-integration-tests/src/test/resources/com/epam/test_generator/api/crud/AsAnAdmin.feature [hunk 4]
```diff
@@ -0,0 +1,23 @@
+Feature: AsAnAdmin scenarios
+
+  @crud-6-1
```
Comment (pomkine) [inline jdi-cucumber-test-generator-integration-tests/src/test/resources/com/epam/test_generator/api/crud/AsAnAdmin.feature]: Please remove annotations as we are not using it for now

File: jdi-cucumber-test-generator-integration-tests/src/test/java/com/epam/test_generator/api/steps/utils/Basics.java (hunk 2 above)
Comment (nikandtos) [reply jdi-cucumber-test-generator-integration-tests/src/test/java/com/epam/test_generator/api/steps/utils/Basics.java]: For access to ObjectMapper

**Pull request merged**